nexus/pkg/artifacts/name_utils.go

```diff
@@ -0,0 +1,27 @@
+package artifacts
+
+import (
+	"fmt"
+	"strings"
+)
+
+func parseArtifactQualifiedName(name string) (entityName string, projectName string, artifactName string, rerr error) {
```

Comment:
Nit: we can pass the artifact id instead of name from Python and then won't need this at all.

Reply:
good point!